docs/faq/data-loading.md

```diff
@@ -22,6 +22,13 @@ Once all this is done, Airbyte resumes your sync from where it left off.
 
 We truly appreciate any contribution you make to help the community. Airbyte will become the open-source standard only if everybody participates.
 
+## **Can Airbyte support 2-way sync i.e. changes from A go to B and changes from B go to A?**
+
+We actually do not support this right now. There are some details around how we handle schema and tables names that isn't going to work for you in the current iteration.
```

Comment:
Instead of saying "we" we should use "Airbyte" -- the emphasis should be on the product :) 

```suggestion
We actually do not support this right now. There are some details around how we handle schema and tables names that isn't going to work for you in the current iteration.
```